pydy/viz/scene.py

```diff
@@ -24,24 +25,20 @@
 
 try:
     import IPython
-    from IPython.html import widgets
-    from IPython.display import display, Javascript
-
-    ipy_ver = parse_version(IPython.__version__)
-    if ipy_ver < parse_version('2.0'):
-        raise ImportWarning('PyDy supports IPython >= 2.0.0, while you have ' +
-                            'IPython ' + IPython.__version__ + ' installed. ' +
-                            'IPython related functionalities will not be ' +
-                            'available')
-        ipython_less_than_3 = None
-    elif ipy_ver >= parse_version('2.0') and ipy_ver < parse_version('3.0'):
+    if parse_version(IPython.__version__) < parse_version('3.0'):
+        msg = ('PyDy only supports IPython >= 3.0.0. You have '
+               'IPython {} installed. IPython related functionalities will '
+               'not be available')
+        with warnings.catch_warnings():
```

Comment:
In Python 3, the filter is only applied in the scope of the ```with warnings.catch_warnings()``` block. 

Here's a short example:
```
import warnings

def func():
    with warnings.catch_warnings():
        warnings.filterwarnings('once'):
        warnings.warn('asd', ImportWarning)
        warnings.warn('asd', ImportWarning)

for i in range(2):
    func()
```

If you run with Python 2 and Python 3, it prints the warning a different number of times.